package.json

```diff
@@ -32,6 +32,7 @@
     "mocha": "^3.0.0",
     "mocha-eslint": "^3.0.1",
     "quick-temp": "^0.1.3",
+    "sinon": "^1.17.7",
```

Comment:
Punting `console.log` work here seems fine (we have been logging directly to `console` for a long time AFAICT).